@xen-orchestra/web-core/lib/components/table/UiTable.vue

```diff
@@ -0,0 +1,63 @@
+<template>
+  <table :class="{ 'vertical-border': verticalBorder, error: color === 'error' }" class="ui-table typo p2-regular">
+    <slot />
+  </table>
+</template>
+
+<script lang="ts" setup>
+import { provide } from 'vue'
+
+const props = defineProps<{
+  color?: 'error'
+  name?: string
+  verticalBorder?: boolean
+}>()
+
+provide('tableName', props.name)
+</script>
+
+<style lang="postcss" scoped>
+.ui-table {
+  width: 100%;
+  border-spacing: 0;
+  background-color: var(--background-color-primary);
```

Comment:
Based on the comment on line 20:
```suggestion
  background-color: var(--background-color);
```